### DISCUSSION TITLE
Why does zsh-autocomplete always show directories before typing?

### DISCUSSION THREAD

**Discussion by nickshanks347** [marlonrichert/zsh-autocomplete#524] [Q&A] [answered]
Even when terminal input is blank, zsh-autocomplete always shows directories:

<img width="242" alt="Screenshot 2022-12-12 at 02 50 33" src="https://user-images.githubusercontent.com/11299982/206950388-116acb06-8a1b-4511-8b34-58755fecd882.png">

If the terminal is blank, I don't want anything to be shown

How can I turn this behaviour off?

**Comment by marlonrichert** [comment, marked as answer]
See #507.
